config/config.toml.example

```diff
@@ -38,6 +38,9 @@ oom-action = "log"
 # Enable coprocessor streaming.
 enable-streaming = false
 
+# Enable local latches for transactions.
```

Comment:
Add description about under what circumstances should user consider turn the feature on.